aladdin.sh

```diff
@@ -373,6 +378,9 @@ while [[ $# -gt 0 ]]; do
         --skip-prompts)
             SKIP_PROMPTS=true
         ;;
+        --no-install-software)
```

Comment:
@jcwilson - I can get this value also from aladdin config like I do for MANAGE_MINIKUBE is a separate PR.
Will push that commit tomorrow.

Reason: Choosing to not let aladdin do config management seems more permanent decision .. similar to choosing MANAGE_MINIKUBE = false